Source/UnitTests/UnitTests.vcxproj

```diff
@@ -54,6 +54,7 @@
     <ClCompile Include="Common\FloatUtilsTest.cpp" />
     <ClCompile Include="Common\MathUtilTest.cpp" />
     <ClCompile Include="Common\NandPathsTest.cpp" />
+    <ClCompile Include="Common\SettingsHandlerTest.cpp" />
```

Comment:
The test must also be added to CMakeLists.txt, otherwise it won't run on platforms other than Windows.

Reply:
Done.